### DISCUSSION TITLE
Collection of discussion points for maintainer KickOff-Meetings

### DISCUSSION THREAD

**Discussion by SaschaMann** [exercism/v3#159] [General] [answered]
Write up from an [AMA question](https://exercism-team.slack.com/archives/CR91YFNG3/p1579275311051100) on Slack. Feel free to edit the post to add more points. I've done some minor edits and added some points of my own. Thanks to @ErikSchierboom, @neenjaw and @iHiD for answering my question during the AMA :)

**These are merely suggestions, you don't have to discuss these points.**

---

## Knowledge management

- Collect resources that could be convenient while developing the track.
- Can we identify existing language pathways to model language concepts? (e.g. many languages have a getting started guide)

## Organisation

- Make sure everyone understands things clearly and is on the same page. Raise open questions and answer them (if possible).
- Decide which Concept Exercise each person wants to start working on (choose X low-hanging fruit of basic types).
- Get to know each other a little bit.
- Decide if tools on top of issues should be used for planning, e.g. project boards.
- Decide if people want to set a motivational deadline for the task they have chosen.
- Decide if there should be a synchronous follow-up meeting.
- If there aren't enough maintainers, collect some ideas on where and how to find more.
- If there aren't enough maintainers, are there any tracks to collaborate with on certain or all concept exercises?

**Comment by neenjaw** [comment]
Nice write-up @SaschaMann!

**Comment by ErikSchierboom** [comment, marked as answer]
Yes, excellent write-up! Thanks.
